src/modules/Account/Devices.ts

```diff
@@ -283,6 +298,36 @@ class Devices extends TagoIOModule<GenericModuleParams> {
     return result;
   }
 
+  /**
+   * ! Waiting for back-end to implement this endpoint
```

Comment:
If there's an issue for this on the back end, maybe we should add a checklist to enable this once released